GeneralsMD/Code/Tools/wdump/chunk_d.cpp

```diff
@@ -603,65 +592,63 @@ void ChunkTableClass::List_W3D_CHUNK_DAMAGE_VERTICES(ChunkItem *Item, CListCtrl
 
 void ChunkTableClass::List_W3D_CHUNK_DAMAGE_COLORS(ChunkItem *Item, CListCtrl *List) {
 
-	struct W3dMeshDamageColorStruct *data;
-	data = (W3dMeshDamageColorStruct *) Item->Data;
+	W3dMeshDamageColorStruct *data = (W3dMeshDamageColorStruct *) Item->Data;
 	int Counter = 0;
-	int index = 0;
 	void *max = (char *) Item->Data + Item->Length;
+	assert(Item->Length % sizeof(W3dMeshDamageColorStruct) == 0);
```

Comment:
To make this code a bit more robust against refactors, could also write `sizeof(*data)`